charts/images.yaml

```diff
@@ -35,12 +35,12 @@ images:
 - name: csi-snapshotter
   sourceRepository: https://github.com/kubernetes-csi/external-snapshotter
   repository: k8s.gcr.io/sig-storage/csi-snapshotter
-  tag: v2.1.5
+  tag: v3.0.3
   targetVersion: ">= 1.17"
 - name: csi-snapshot-controller
   sourceRepository: https://github.com/kubernetes-csi/external-snapshotter
   repository: k8s.gcr.io/sig-storage/snapshot-controller
-  tag: v2.1.5
+  tag: v3.0.3
```

Comment:
I think we also need to specify the targetVersion `>=1.17` for `snapshot-controller` here